nucleus/core/kernel/src/main/java/com/sun/enterprise/v3/server/ApplicationLifecycle.java

```diff
@@ -577,39 +577,37 @@ public Types getDeployableTypes(DeploymentContext context) throws IOException {
             Types types = context.getTransientAppMetaData(Types.class.getName(), Types.class);
             if (types!=null) {
                 return types;
-            } else {
```

Comment:
Wanted to address the resource issue but its unclear to me if that is correct since the parser is passed to something outliving the context of this method so all this change is is the removal or unnecessary `else` clause I did when looking at it.

Reply:
Added a suppression.